spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -11,6 +11,12 @@
   let(:success_with_liveness_response) do
     instance_double(Faraday::Response, status: 200, body: success_with_liveness_response_body)
   end
+  let(:failure_face_match_fail_response_body) do
+    LexisNexisFixtures.true_id_response_with_face_match_fail
+  end
+  let(:failure_response_face_match_fail) do
+    instance_double(Faraday::Response, status: 200, body: failure_face_match_fail_response_body)
+  end
```

Comment:
```suggestion
  let(:failure_response_face_match_fail) do
    instance_double(Faraday::Response, status: 200, body:     LexisNexisFixtures.true_id_response_with_face_match_fail)
  end
```

seems like we could opt to condense this?

Reply:
That's a good point. I was following the pattern in the file, but maybe this is a good chance to condense more of it. I'll timebox doing that.

Reply:
@amirbey I made the change in [32469c1](https://github.com/18F/identity-idp/pull/9937/commits/32469c14e753d9be191ad5378fffb86aed70dd01). I know you approved the PR but since that commit is a decently big test change, if you don't mind I'd appreciate you taking a look before I merge.